src/components/EntityPage/EntityScorecardDetails/EntityScorecardDetails.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2021 Cortex Applications, Inc.
```

Comment:
super nit: idk why the eslint-config didn't update to 2022, could we update these?